src/app/campaign-list/campaign-card.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, ChangeDetectionStrategy } from '@angular/core';
+import { Campaign } from './campaign-list.service';
+
+@Component({
+  selector: 'grove-campaign-card',
+  template: `
+    <div class="header {{campaign?.status}}"></div>
+    <section *ngIf="campaign">
+      <div>{{campaign.flights | campaignFlightDates}}</div>
+      <h2>
+        <a routerLink="{{'/campaign/' + campaign.id}}">
+          {{campaign.advertiser && campaign.advertiser.name}}
+        </a>
+      </h2>
+      <div>
+        <span class="status {{campaign.status}}">{{campaign.status}}</span>
+        {{campaign.type | campaignType}}
+      </div>
+      <div *ngIf="campaign.flights | campaignFlightTargets as targets">
+        <prx-icon size="1em" name="globe"></prx-icon>
+        {{targets}}
+      </div>
+      <div>{{campaign.flights | campaignFlightZones}}</div>
+    </section>
+    <section class="loading" *ngIf="!campaign">
+      <prx-spinner></prx-spinner>
+    </section>
+  `,
+  styleUrls: ['campaign-card.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
`changeDetection: ChangeDetectionStrategy.OnPush` component should only update with change detection when @Input property changes